aspnetcore/fundamentals/environments.md

```diff
@@ -125,84 +127,93 @@ The production environment should be configured to maximize security, performanc
 
-**Command line**
-```
+To set the `ASPNETCORE_ENVIRONMENT` for the current session when the app is started using [dotnet run](/dotnet/core/tools/dotnet-run), the following commands are used:
+
+**Command prompt**
+
+```console
 set ASPNETCORE_ENVIRONMENT=Development
 ```
+
 **PowerShell**
-```
+
+```powershell
 $Env:ASPNETCORE_ENVIRONMENT = "Development"
 ```
 
-These commands take effect only for the current window. When the window is closed, the ASPNETCORE_ENVIRONMENT setting reverts to the default setting or machine value. In order to set the value globally on Windows open the **Control Panel** > **System** > **Advanced system settings** and add or edit the `ASPNETCORE_ENVIRONMENT` value.
+These commands only take effect for the current window. When the window is closed, the `ASPNETCORE_ENVIRONMENT` setting reverts to the default setting or machine value. In order to set the value globally in Windows, open the **Control Panel** > **System** > **Advanced system settings** and add or edit the `ASPNETCORE_ENVIRONMENT` value:
 
 ![System Advanced Properties](environments/_static/systemsetting_environment.png)
 
 ![ASPNET Core Environment Variable](environments/_static/windows_aspnetcore_environment.png)
 
-
 **web.config**
 
 See the *Setting environment variables* section of the [ASP.NET Core Module configuration reference](xref:host-and-deploy/aspnet-core-module#setting-environment-variables) topic.
 
 **Per IIS Application Pool**
 
-To set environment variables for individual apps running in isolated Application Pools (supported on IIS 10.0+), see the *AppCmd.exe command* section of the [Environment Variables \<environmentVariables>](/iis/configuration/system.applicationHost/applicationPools/add/environmentVariables/#appcmdexe) topic.
+To set environment variables for individual apps running in isolated Application Pools (supported on IIS 10.0+), see the *AppCmd.exe command* section of the [Environment Variables &lt;environmentVariables&gt;](/iis/configuration/system.applicationHost/applicationPools/add/environmentVariables/#appcmdexe) topic.
 
 ### macOS
-Setting the current environment for macOS can be done in-line when running the application;
+
+Setting the current environment for macOS can be performed in-line when running the app:
 
 ```bash
 ASPNETCORE_ENVIRONMENT=Development dotnet run
 ```
-or using `export` to set it prior to running the app.
+
+Alternatively, set the environment with `export` prior to running the app:
 
 ```bash
 export ASPNETCORE_ENVIRONMENT=Development
 ```
-Machine level environment variables are set in the *.bashrc* or *.bash_profile* file. Edit the file using any text editor and add the following statment.
+
+Machine-level environment variables are set in the *.bashrc* or *.bash_profile* file. Edit the file using any text editor and add the following statment:
```

Comment:
- statment --> statement
- Consider adding a comma before "and add the following statement", since that part of the sentence can stand on its own and still make sense.

Reply:
* Spelling 🤦‍♂️ :smile:
* Compound verb, so no comma since I didn't repeat the assumed "you" subject. However, let's break it ...

> Edit the file using any text editor. Add the following statement: